ReleaseNotes.md

```diff
@@ -57,8 +57,13 @@ categorized by license:
 
 ### BSD-3
 
+* [Hypher v0.2.5](https://github.com/bramstein/Hypher)
 * [Sinon v6.1.3](http://sinonjs.org)
 
+### LGPL
```

Comment:
@colinbdclark - can we manage this?

Reply:
Yup, LGPL is not a problem—especially if we're managing this dependency via npm, rather than including it directly in our source tree.